src/uct/ib/ud/base/ud_iface.h

```diff
@@ -381,9 +385,27 @@ uct_ud_iface_get_async_time(uct_ud_iface_t *iface)
     return iface->super.super.worker->async->last_wakeup;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_time_t
+uct_ud_iface_get_time(uct_ud_iface_t *iface)
+{
+    return ucs_get_time();
+}
```

Comment:
can we remove this wrapper?

Reply:
don't we want some logic caching ucs_get_time() calls?

Reply:
i don't see such logic in this function..

Reply:
also, need to be careful with it, since now timer resolution is higher